web-test-runner.config.js

```diff
@@ -84,7 +84,7 @@ export default {
     protocol: 'https:',
     concurrency: 4,
     concurrentBrowsers: 1,
-    testsFinishTimeout: 30000,
+    testsFinishTimeout: 60000,
```

Comment:
Does seem to effect actual runtime, but we should try to look into this when next we have a lot of extra time available... 😏